pkg/apis/options/options.go

```diff
@@ -45,6 +45,7 @@ type Options struct {
 	BitbucketRepository      string   `flag:"bitbucket-repository" cfg:"bitbucket_repository"`
 	EmailDomains             []string `flag:"email-domain" cfg:"email_domains"`
 	WhitelistDomains         []string `flag:"whitelist-domain" cfg:"whitelist_domains"`
+	ExtraHeaders             []string `flag:"extra-header" cfg:"extra_header"`
```

Comment:
This should be extra_headers (plurarl) for the config file to be consistent
```suggestion
	ExtraHeaders             []string `flag:"extra-header" cfg:"extra_headers"`
```

Reply:
Ok. Done